### DISCUSSION TITLE
Differentiate the client's recieved messages in websocket

### DISCUSSION THREAD

**Discussion by torajghaffari** [beyondcode/laravel-websockets#1100] [General]
**Hi everyone**
I'm newbe at websocket and I want to implementing a chat application api with features like last seen of users (online-last seen time)
or unicasting instead of broadcasting .

My problem is how to differentiate the client's recieved messages in websocket like online rooms or new messages of each room

 anybody have a good sample or a documantion for that ?

**Comment by maxcelos** [comment]
Your question is more about Laravel Broadcasting concepts. You should look into documentation or search for examples of broadcasting in general not specific for this package.
